src/site/content/en/blog/top-cwv-2023/index.md

```diff
@@ -0,0 +1,203 @@
+
+The Lighthouse audit [Avoid non-composited animations](https://developer.chrome.com/docs/lighthouse/performance/non-composited-animations/) will warn when a page animates potentially slow CSS properties, a large number of which can cause layout shifts.
+
+## First Input Delay (FID)
+
+Our last set of recommendations are for [First Input Delay (FID)](/fid/), which is a measure of a page's responsiveness to user interactions. While most sites on the web currently [score very well](https://datastudio.google.com/s/vax9YUKhRN0) on FID, we've [documented](/better-responsiveness-metric/#what-improvements-are-we-considering) shortcomings of the FID metric in the past, and we believe there is still a lot of opportunity for sites to improve their overall responsiveness to user interactions.
+
+Our new [Interaction to Next Paint (INP)](/inp/) metric is a possible successor to FID, and all of the recommendations below apply equally well to both FID and INP. Given that sites [perform worse](https://almanac.httparchive.org/en/2022/performance#inp-as-a-hypothetical-cwv-metric) on INP than FID, especially on mobile, we encourage developers to seriously consider these responsiveness recommendations, despite having "good" FID.
+
+### Avoid or break up long tasks
+
+Tasks are any piece of discrete work that the browser does. Tasks include rendering, layout, parsing, and compiling and executing scripts. When tasks become [long tasks](/long-tasks-devtools/#what-are-long-tasks)—that is, 50 milliseconds or longer—they block the main thread from being able to respond quickly to user inputs.
+
+Per the Web Almanac, there's [plenty of evidence](https://almanac.httparchive.org/en/2022/javascript#long-tasksblocking-time) to suggest that developers could be doing more to avoid or break up long tasks. While breaking up long tasks may not be as low of an effort as other recommendations in this article, it's less effort than other techniques not offered in this article.
+
+While you should always strive to do as little work as possible in JavaScript, you can help the main thread quite a bit by [breaking up long tasks into smaller ones](/optimize-long-tasks/). You can accomplish this by **[yielding to the main thread](/optimize-long-tasks/#use-asyncawait-to-create-yield-points)** often so that rendering updates and other user interactions can occur more quickly.
+
+Another option is to consider using APIs such as `isInputPending` and the Scheduler API. [`isInputPending`](/optimize-long-tasks/#yield-only-when-necessary) is a function that returns a boolean value that indicates whether a user input is pending. If it returns true, you can yield to the main thread so it can handle the pending user input. [The Scheduler API](/optimize-long-tasks/#a-dedicated-scheduler-api) is a more advanced approach, which allows you to schedule work based on a system of priorities that take into account whether the work being done is user-visible or backgrounded.
```

Comment:
```suggestion
Another option is to consider using APIs such as `isInputPending` and the Scheduler API. [`isInputPending`](/optimize-long-tasks/#yield-only-when-necessary) is a function that returns a boolean value that indicates whether a user input is pending. If it returns `true`, you can yield to the main thread so it can handle the pending user input.

The [Scheduler API](/optimize-long-tasks/#a-dedicated-scheduler-api) is a more advanced approach, which allows you to schedule work based on a system of priorities that take into account whether the work being done is user-visible or backgrounded.
```

Reply:
Done